src/scripts/main.js

```diff
@@ -21,7 +21,37 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul1Lvl = document.createElement('ul');
+
+  element.append(ul1Lvl);
+
+  for (const key1 in data) {
+    const li1Lvl = document.createElement('li');
+    const ul2Lvl = document.createElement('ul');
+
+    li1Lvl.textContent = key1;
+    ul1Lvl.append(li1Lvl);
+    li1Lvl.append(ul2Lvl);
+
+    for (const key2 in data[key1]) {
+      const li2Lvl = document.createElement('li');
+      const ul3Lvl = document.createElement('ul');
+
+      ul2Lvl.append(li2Lvl);
+      li2Lvl.textContent = key2;
+
+      if (li2Lvl.textContent !== 'Wine' && li2Lvl.textContent !== 'Schnaps') {
```

Comment:
You duplicate the logic a few times and hardcoded code for special keys. Instead, let's do a universal solution using recursive calls

You can check if `data[key]` is an object and if yes - call `createTree` for it again